go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/spf13/cobra v1.7.0
 	github.com/stretchr/testify v1.8.4
 	github.com/vmware-tanzu/net-operator-api v0.0.0-20210401185409-b0dc6c297707
-	github.com/vmware-tanzu/vm-operator/api v0.0.0-20221204041501-ec03d8bb733c
+	github.com/vmware-tanzu/vm-operator/api v0.0.0-20230717155710-8f2552faf4dc
```

Comment:
Update to https://github.com/vmware-tanzu/vm-operator/releases/tag/v1.8.2

Reply:
It doesn't seem what have a tagged version for VM operator API module
https://pkg.go.dev/github.com/vmware-tanzu/vm-operator/api

Reply:
Yeah looks like in general there are no api/ tags on the vm operator repo

Reply:
Ack. Have asked them push the api/v1.8.2 tag as well.